utils/zerovec/src/varzerovec/vec.rs

```diff
@@ -254,6 +254,15 @@ impl<'a, T: VarULE + ?Sized> VarZeroVec<'a, T> {
         Ok(VarZeroVec::Borrowed(borrowed))
     }
 
+    /// Uses a potentially invalid bytes slice as a `VarZeroVec`.
```

Comment:
Nit: this is confusing since the slice can't be invalid. Change this to the "without verification" prose you have on ZeroVec

Reply:
How can it not be invalid? If every slice was valid this wouldn't be unsafe

Reply:
The function doesn't return a result and it doesn't panic; it assumes the bytes are valid. It's undefined behavior to call it with invalid bytes, so you shouldn't document that it accepts "potentially invalid bytes"

Reply:
I'm just saying that you have two versions of this doc string,

```
    /// Uses a `&[u8]` buffer as a `ZeroVec<T>` without any verification.
    ///
    /// # Safety
    ///
    /// `bytes` need to be an output from [`ZeroSlice::as_bytes()`], and `size_of`
    /// is `mem::size_of::<T::ULE>()`.
```

and

```
    /// Uses a potentially invalid bytes slice as a `VarZeroVec`.
    ///
    /// # Safety
    ///
    /// `bytes` need to be an output from [`VarZeroSlice::as_bytes()`].
```

and I find the `ZeroVec` version clearer and less contradictory.